sympy/solvers/bivariate.py

```diff
@@ -113,6 +114,28 @@ def _linab(arg, symbol):
         x = -x
     return a, b, x
 
+def power_list(f, *gens, **args):
+    """
+    Helper function to return all powers of variables
+    present in f.
```

Comment:
This doesn't makes sense. All powers of variables in what order?

Reply:
Yeah It had to be improved. This is not the final code of powerlist.